NEWS.md

```diff
@@ -1,4 +1,18 @@
-## 8.0.0 - Unreleased
+## 9.0.0 - Unreleased
+
+## 8.0.0 - Released
+The main focus of this release was to implement API for order templates and to clean-up API for acquisition-unit-assignments.
+
+[Full Changelog](https://github.com/folio-org/mod-orders-storage/compare/v7.0.0...v8.0.0)
+
+### Stories
+* [MODORDSTOR-106](https://issues.folio.org/browse/MODORDSTOR-106) - Create order-templates API
```

Comment:
https://github.com/folio-org/mod-orders-storage/blob/00db9642f576efefdce5bdf696256a9123f2a53b/src/main/resources/templates/db_scripts/schema.json#L363
Also please change this to 8

Reply:
Updated.